### PR TITLE
Improving Kaiaulu documentation

### PR DESCRIPTION
**Pull request by nicoelee123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ If you are not familiar with R, you can find command line interface scripts in t
     * Issue ID parsers from commit messages 
     * Name and E-mail Identity Matching (For identifying same users inter and intra GitLog, Mailing List, etc.)
  * [Parser to Network Transformations](http://itm0.shidler.hawaii.edu/kaiaulu/reference/index.html#section-networks) + Interactive visualizations (See Getting Started below)
-    * Graph representation uses node + edge format, making easier to extend analysis in popular graph tools such as Gephi and igraph library. 
+    * Graph representation uses node + edge format, making it easier to extend analysis in popular graph tools such as Gephi and igraph library.
```
Comment (nicoelee123) [inline README.md:30]: Makes it grammatically correct and easier to follow

File: vignettes/blamed_line_types_showcase.Rmd [hunk 2]
```diff
@@ -28,7 +28,7 @@ This vignette introduces the blamed line types concept of Kaiaulu. This is an ex
 
 ## Project Configuration File
 
-To identify line types, we must choose what types of lines are of our interest (e.g. functions, structs, etc). Kaiaulu defers to the project configuration file the choice of line types and uses [Universal Ctags](https://github.com/universal-ctags/ctags). The relevant portion of the configuration file in APR is as follows:
+To identify line types, we must choose what types of lines are of our interest (e.g. functions, structs, etc). Kaiaulu chooses the line types from the project configuration file and uses [Universal Ctags](https://github.com/universal-ctags/ctags). The relevant portion of the configuration file in APR is as follows:
```
Comment (nicoelee123) [inline vignettes/blamed_line_types_showcase.Rmd:31]: Easier to follow

File: vignettes/blamed_line_types_showcase.Rmd [hunk 3]
```diff
@@ -57,7 +57,7 @@ tool:
         - f # functions
 ```
 
-[Kaiaulu's GitHub wiki page](https://github.com/sailuh/kaiaulu/wiki/Universal-Ctags) also provides more details on Universal Ctags various flags.
```
Comment (nicoelee123) [inline vignettes/blamed_line_types_showcase.Rmd:60]: Grammatically correct

File: vignettes/bug_count.Rmd [hunk 4]
```diff
@@ -51,7 +51,7 @@ file_extensions <- conf[["filter"]][["keep_filepaths_ending_with"]]
...
+To establish bug count, we must map each file to its associated issue. In general (but not always), an open source project will adopt a commit message convention to label issue ids. For example, Kaiaulu uses `i #<issue number>`. JIRA dictates issue ids in the format PROJECT-<issue-number>. We can use this assumption to find the issues that the git log commits are trying to fix. You can create your own regular expression by manually inspecting some of the commit messages (for example, compare the regular expression from the geronimo project used here against [its commit message](https://github.com/apache/geronimo/commits/trunk)).
```
Comment (nicoelee123) [inline vignettes/bug_count.Rmd:54]: Easier to understand

File: vignettes/bug_count.Rmd [hunk 5]
```diff
@@ -146,7 +146,7 @@ project_git <- merge(project_git,jira_issues,all.x=TRUE,by.x="commit_message_id"
                      by.y="issue_key")
 ```
 
-With the added information per commit per file of which commits were associated to issues which were bugs, we can then group by filepath, the number of issues which the type is bug. The following show the top 20 buggiest files:
+For each file, we now know the commits that were associated with fixing bugs. We then group the number of bug-related issues by file path. Our result should show the number of bugs per file path. The following outputs the top 20 files with the most bugs.
```
Comment (nicoelee123) [inline vignettes/bug_count.Rmd:149]: Easier to understand

File: vignettes/community_detection_showcase.Rmd [hunk 6]
```diff
@@ -35,7 +35,7 @@ This notebook showcases the use of community detection using OSLOM [1], which re
...
+To facilitate the usage of the OSLOM algorithm, Kaiaulu currently provides two interface functions to its two binaries: `community_oslom` and `recolor_network_by_community`. The former requires the binary path to OSLOM to be provided and parses OSLOM outputs into R memory as a list of tables closest to the raw output. The latter leverages said list of tables to re-color a network by its assigned community up to 12 different communities (the limitation is only on color, the OSLOM output will give as many communities as detected). Nodes that are assigned to more than one community are colored black.
```
Comment (nicoelee123) [inline vignettes/community_detection_showcase.Rmd:38]: Misspelling

**Review by carlosparadis — CHANGES_REQUESTED** [review]
Hi,

These look fine. Could you git squash then into a single commit? 

Thanks!

**Comment by nicoelee123** [comment]
Hello,

I think I successfully combined the commits into a single commit. Please let me know if I didn't do it correctly. 

Thank you

**Pull request merged**